core/src/com/po/fuck/FUCK.java

```diff
@@ -1,67 +1,62 @@
 import com.badlogic.gdx.Gdx;
-import com.badlogic.gdx.graphics.OrthographicCamera;
-import com.badlogic.gdx.graphics.g2d.SpriteBatch;
 import com.badlogic.gdx.math.Vector2;
-import com.badlogic.gdx.utils.ScreenUtils;
-import com.po.fuck.enemies.BasicEnemy;
 import com.po.fuck.lifetime.Destructable;
 import com.po.fuck.lifetime.Managed;
 import com.po.fuck.lifetime.Manager;
 import com.po.fuck.collections.All;
 
-import static com.po.fuck.Constants.GAME_HEIGHT;
-import static com.po.fuck.Constants.GAME_WIDTH;
-
 public class FUCK extends ApplicationAdapter {
     static {
-       forceInit(Destructable.class);
-       forceInit(All.class);
+        forceInit(Destructable.class);
+        forceInit(All.class);
     }
     public static Managed<Player> player;
-
-    SpriteBatch batch;
-    private OrthographicCamera camera;
+    public static Managed<Camera> camera;
 
     @Override
     public void create() {
-        camera = new OrthographicCamera();
-        camera.setToOrtho(false, GAME_WIDTH, GAME_HEIGHT);
-        batch = new SpriteBatch();
-
-        player = Manager.create(new Player(new Vector2(GAME_WIDTH / 2, GAME_HEIGHT / 2)));
-
-        Manager.create(new BasicEnemy(new Vector2(100, 100)));
-        Manager.create(new BasicEnemy(new Vector2(GAME_WIDTH - 100, GAME_HEIGHT - 100)));
+        camera = Manager.create(new Camera());
+        player = Manager.create(new Player(new Vector2()));
+
+        Manager.create(new Room(new Vector2(0, 0)));
+        Manager.create(new Room(new Vector2(1, 0)));
+
+        // Creating some game borders to destroy the bullets that went off the map.
+        // We can not use here VERTICAL/HORIZONTAL, because if something went off the
+        // map, then we want to catch with a thick wall, because it can be laggy or
+        // something else.
+        final int HUGE = 1000, HUGE2 = 2 * HUGE, SIZE = GAME_BORDER + HUGE2;
+        Manager.create(new InvisibleWall(new Vector2(-GAME_BORDER - HUGE, 0), new Vector2(HUGE2, SIZE)));
+        Manager.create(new InvisibleWall(new Vector2(-GAME_BORDER + HUGE, 0), new Vector2(HUGE2, SIZE)));
+
+        Manager.create(new InvisibleWall(new Vector2(0, -GAME_BORDER - HUGE), new Vector2(SIZE, HUGE2)));
+        Manager.create(new InvisibleWall(new Vector2(0, -GAME_BORDER + HUGE), new Vector2(SIZE, HUGE2)));
     }
 
     @Override
     public void render() {
-        ScreenUtils.clear(0, 0, 0, 1);
-        camera.update();
-        batch.setProjectionMatrix(camera.combined);
+        if (player.get() == null)
+            player = Manager.create(new Player(new Vector2()));
+
+        camera.get().setPosition(player.get().getPosition());
 
         float delta = Gdx.graphics.getDeltaTime();
         All.updatableCollection.update(delta);
 
-        batch.begin();
-        All.drawableCollection.draw(new CenterDrawer(batch));
-        batch.end();
-    }
-
-    @Override
-    public void dispose() {
-        batch.dispose();
+        camera.get().draw_all();
```

Comment:
What?

Reply:
What is wrong ? Draw is not in dispose if you are thinking about it, look for the whole code

Reply:
Stupid me and misleading github